src/test/end-to-end-tests/test_suites/play_test.ts

```diff
@@ -0,0 +1,228 @@
+            requiresVoiceConnection: true,
+        },
+        {
+            command: "blackpink", // ddu du ddu du - blackpink
+            responseValidator: (
+                title: string,
+                description: string,
+                parsedGameOptions?: ParsedGameOptionValues,
+                client?: Eris.Client,
+            ) => description.includes("guessed correctly"),
+            expectedResponseType: KmqResponseType.RAW,
+            requiresVoiceConnection: true,
+        },
+        {
+            command: ",guessmode both",
+            responseValidator: (
+                title: string,
+                description: string,
+                parsedGameOptions?: ParsedGameOptionValues,
+                client?: Eris.Client,
+            ) =>
+                parsedGameOptions!["guessmode set"]!.value === "both" &&
+                parsedGameOptions!["guessmode set"]!.updated,
+            expectedResponseType: KmqResponseType.GAME_OPTIONS_RESPONSE,
+            requiresVoiceConnection: true,
+        },
+        {
+            command: "blackpink", // kill this love - blackpink
+            responseValidator: (
+                title: string,
+                description: string,
+                parsedGameOptions?: ParsedGameOptionValues,
+                client?: Eris.Client,
+            ) => description.includes("guessed correctly"),
+            expectedResponseType: KmqResponseType.RAW,
+            requiresVoiceConnection: true,
+        },
+        {
+            command: "dynamite", // dynamite - bts
+            responseValidator: (
+                title: string,
+                description: string,
+                parsedGameOptions?: ParsedGameOptionValues,
+                client?: Eris.Client,
+            ) => description.includes("guessed correctly"),
+            expectedResponseType: KmqResponseType.RAW,
+            requiresVoiceConnection: true,
+        },
+        {
+            command: "boy with luv", // boy with luv - bts, now with a guess streak
+            responseValidator: (
+                title: string,
+                description: string,
+                parsedGameOptions?: ParsedGameOptionValues,
+                client?: Eris.Client,
+            ) =>
+                description.includes("guessed correctly") &&
+                description.includes("(🔥5)"),
+            expectedResponseType: KmqResponseType.RAW,
+            requiresVoiceConnection: true,
+        },
+        {
+            command: ",skip",
+            responseValidator: (
+                title: string,
+                description: string,
+                parsedGameOptions?: ParsedGameOptionValues,
+                client?: Eris.Client,
+            ) => true,
+            expectedResponseType: KmqResponseType.RAW,
+            requiresVoiceConnection: true,
+        },
+        {
+            command: "gangnma style", // gangnam style - psy, lost guess streak, unique songs reset, allow typo
```

Comment:
Do we already have a command that sets it to typingtypos elsewhere? Is it a part of the other test file?

Reply:
It's the default